r/data-raw/docgen.R

```diff
@@ -149,6 +149,10 @@ tidyselect <- grep("^tidyselect::", readLines("R/reexports-tidyselect.R"), value
 # HACK: remove the _random_along UDF we're using (fix in ARROW-17974)
 docs[["_random_along"]] <- NULL
 
+# TODO - update the script to add this back in - will fail CI as tries to link
+# to non-existent function as arrow::one only exists as registered binding
+docs[["arrow::one"]] <- NULL
+
```

Comment:
Do we need to do this now? Or is this a thing for later? If later, do we have an issue for it? I'm not sure I fully get what the comment is saying (but will admit I haven't dug too much right now)

Reply:
So without it we were getting the warning shown here: https://github.com/apache/arrow/actions/runs/15282844766/job/42985877041#step:5:612

I don't think we need to do it now, I will open a ticket.